### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/guacamole-composition.git guacamole-composition
    cd guacamole-composition
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS .github/workflows/build.yml .gitignore README.md docker-compose.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by mcdonnnj** [comment]
Accidentally pushed before committing.

**Review by dav3r — APPROVED** [review]
🍏

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,13 @@
+# This file specifies intentionally untracked files that Git should ignore.
+# Files already tracked by Git are not affected.
+# See: https://git-scm.com/docs/gitignore
+
+## Project Specific ##
```
Comment (jsf9k) [inline .gitignore:5]: Not a big deal if you choose to ignore this suggestion.
```suggestion
## Project-specific ##
```

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ Connect to the `guacamole` web interface at:
...
 as soon as possible.
 
+### Volumes ###
+
+#### postgres ####
+
+| Mount Point | Purpose |
+| ----------- | ------- |
+| `dbdata` | Stores all database data for the postgres container |
+| `dbinit` | Stores the postgres initialization script for the guacamole database resources |
```
Comment (dav3r) [inline README.md]: There are probably more of these that should be updated to the [preferred capitalization](https://guacamole.apache.org/):
```suggestion
| `dbinit` | Stores the postgres initialization script for the Guacamole database resources |
```

**Review by dav3r — APPROVED** [review]
LGTM, aside from a minor capitalization thang.  🥑

**Review by jsf9k — APPROVED** [review]
LGTM, other than the capitalization jazz that @dav3r pointed out.

File: README.md (hunk 2 above)
Comment (mcdonnnj) [reply README.md]: I adjusted this to better reflect references to `guacamole` the container and Guacamole the software in e89866e. Please let me know if you have any feedback on those changes.

Comment (dav3r) [reply README.md]: LGTM!

**Pull request merged**